release-notes/schema/dotnet-required-packages.json

```diff
@@ -0,0 +1,170 @@
+{
+  "description": "The set of packages required by a given .NET version for a set of distros.",
+  "type": [
+    "object",
+    "null"
+  ],
+  "properties": {
+    "ChannelVersion": {
+      "type": "string"
+    },
+    "EolDate": {
+      "type": "string",
+      "format": "date"
+    },
+    "Packages": {
+      "type": "array",
+      "items": {
+        "description": "A logical package that will exist in various distros with different package names. Includes the scenarios for which the package is required.",
+        "type": [
+          "object",
+          "null"
+        ],
+        "properties": {
+          "Id": {
+            "type": "string"
+          },
+          "Name": {
+            "type": "string"
+          },
```

Comment:
Should we consider the case where functionality in some distros could be split across multiple packages? It's not the best example since it's not really a separate distro, but Ubuntu Chiseled splits packages into multiple slices. However I don't think we've needed to install multiple slices of one package at once yet though.

Reply:
I think that should be fine. I think we already have that case. Yes?

https://github.com/dotnet/core/blob/0babb5dc73c7ecc8e835da8efc8055d5b45d84b6/release-notes/9.0/os-packages.json#L121-L128

Reply:
Didn't notice that before. Looks good!